src/idsk/components/search-results-filter/template.njk

```diff
@@ -41,12 +41,12 @@
             </span>
             <span class="idsk-search-results__date__span__example">Napr. 2005, alebo 8.3.2021
             </span>
-            <input class="govuk-input" type="text" name="{{item.idFrom}}" id="{{item.idFrom}}">
+            <input class="govuk-input" type="text" name="{{item.idFrom}}" id="{{item.idFrom}}" aria-label="Vstupné pole pre zadanie počiatočného dátumu aktualizácie">
```

Comment:
V komponente by nemali byť priamo vložené texty, keďže potom by nešli jazykové mutácie. Pozri ako to ma napríklad table-filter (a rovnaké platí pre ostatné časti requestu)